README.md

```diff
@@ -19,6 +19,10 @@ __Database migrations written in Go. Use as [CLI](#cli-usage) or import as [libr
 
 Forked from [mattes/migrate](https://github.com/mattes/migrate)
 
+# Install Latest Version
+
+`go get github.com/golang-migrate/migrate/v4`
```

Comment:
Check https://github.com/golang-migrate/migrate/tree/master/cmd/migrate and https://github.com/golang-migrate/migrate#use-in-your-go-project
This update of Readme is redundant